Scripts/main.js

```diff
@@ -1,15 +1,30 @@
-var brdfViewport = null;
-var modelViewport = null;
-
-var render = function(time) {
-  brdfViewport.render(time);
-  modelViewport.render(time);
-  requestAnimationFrame(render);
-}
-
-/* Requires BRDFViewport.js */
-document.addEventListener('DOMContentLoaded', function () {
-  brdfViewport = new BRDFViewport("brdf-canvas", 512, 512);
-  modelViewport = new ModelViewport("model-canvas", 512, 512);
-  requestAnimationFrame(render);
-});
+"use strict";
```

Comment:
The lines below break on my version of chrome:
I get:
"Failed to load module script: The server responded with a non-JavaScript MIME type of "text/plain". Strict MIME type checking is enforced for module scripts per HTML spec."

Reply:
The most likely reason why things would break is ES6 modules, and a [the first result on a quick Google search seems to confirm this](https://stackoverflow.com/questions/47403478/es6-modules-in-local-files-the-server-responded-with-a-non-javascript-mime-type). In Chrome they should work but if you load the files locally the same-origin policy causes @n8vm 's error message.

I may want to leave ES6 modules out because even though they should work with the latest Chrome, they still don't work out of the box in the latest Firefox (and possibly other browsers)?

Reply:
Hmm, I’m thinking it might have something to do with how i’m hosting my local server. 

How have you been serving the files on your machine? I’ve been using python -m http.server.

Reply:
I am also using `python -m http.server`.